packages/mds-policy/api.ts

```diff
@@ -378,10 +373,10 @@ function api(app: express.Express): express.Express {
     const validation = Joi.validate(policy, policySchema)
     const details = validation.error ? validation.error.details : null
 
-    // FIXME is basically identical to POST policy
+    // TODO is basically identical to POST policy
 
     if (details) {
-      log.info('questionable policy json', details)
+      log.info('policy JSON', details)
```

Comment:
This is an error condition, and should be treated as such (even though this endpoint will likely be cleaned up a bunch in the near future). Should be Perhaps something like `log.error('Invalid Policy JSON', details)'` as the log message.